src/server/pfs/server/testing/server_test.go

```diff
@@ -890,6 +890,33 @@ func TestPFS(suite *testing.T) {
 		require.Equal(t, data, b.String())
 	})
 
+	suite.Run("BasicFileWithOffset", func(t *testing.T) {
```

Comment:
I see "SimpleFile" below, we should be consistent with "Basic" vs. "Simple"

Reply:
One thing to consider here is that when working on changes to the read path you might want to run all the GetFile tests using `go test -run GetFile`.  Calling this `GetFileWithOffset` would let us easily match this with the other GetFile tests.